src/pages/help.js

```diff
@@ -74,7 +76,9 @@ export default function Help(props) {
               align="left"
             />
             <section>
-              <h2>Buy a Course</h2>
+              <h1>
```

Comment:
As an idea, having more than 1 `h1` per page isn't recommended for accessibility reasons. I think this should remain an `h2` and if we have another `h1` somewhere, we should also change it to something else.